stylesheet.css

```diff
@@ -0,0 +1,257 @@
+body {
+  height: 100vh;
+  width: 100vw;
+  margin: 0;
+  background-image: url("images/galaxy.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: center center;
+  font-family: "Space Mono", monospace;
+}
+.header {
+  color: white;
+  text-align: center;
+  font-family: "Space Mono", monospace;
```

Comment:
Font-size is inheritable! You already set this on the body so all the child elements (aka every element) will inherit this value unless otherwise specified.